.emacs.d/init.el

```diff
@@ -109,6 +116,10 @@ Despite this, your config appears to have loaded successfully.")
   (if loaddefs-buffer
       (kill-buffer loaddefs-buffer)))
 
+;; If in dev mode, load the test runner.
+(when kotct/dev-mode
+  (require 'dot-tests "~/.emacs.d/test/lisp/dot-tests"))
```

Comment:
I'm hesitant to include this (both the var and the thing) in `init.el` because
- It adds calls to the `init` stream under normal non-dev circumstances, and
- Adding this to the source tree for the entire configuration means that we're tainting our actual configuration with testing stuff.

I think tests should be run in a separate Emacs instance from the existing one, if the developer is running one.  The ability to run `kotct/run-tests` within the normal running circumstances is somewhat sketchy to me.  Far better to run Emacs using a `--script` that also loads `init` than do this, I think.